src/modules/users/resolvers/user-skill.js

```diff
@@ -0,0 +1,29 @@
+const { schemaComposer } = require('graphql-compose')
+const User = require('../../../models/user')
+
+const UserTC = schemaComposer.getOTC('User')
+
+UserTC.addResolver({
```

Comment:
This is incorrectly implemented. If a qualification relates to a user, a `user` field of type `objectID` should be added to the `qualification` schema.

Reply:
it is something else. it does'nt have to do qualification and user relationship.

Reply:
If you've correctly implemented the relationship between `user` and `relationship`, there's no need for this anymore.

In `/schemas/qualification.js`

```javascript
const mongoose = require('mongoose')
const Schema = require('mongoose')
const uniqueValidator = require('mongoose-unique-validator')

const qualificationsSchema = new mongoose.Schema({
  title: {
    type: String,
    required: true
  },
  type: {
    type: String,
    required: true
  },
  user: {
    type: Schema.Types.ObjectId,
    default: null
  }
})

// plugins
qualificationsSchema.plugin(uniqueValidator)

module.exports = qualificationsSchema
```

Since you've correctly declared the relationship, the resolver is automatically created. You can do:

```javascript
query {
   qualifications(filter: {user: <USER_ID>}) {
     title
     user {
       firstName
       lastName
       email
     }
   }
}
```
This will return all of the qualifications (`skill` and `knowledge`) added to user of `<USER_ID>`